lib/common/analytics/analytics_event_names.dart

```diff
@@ -42,6 +42,7 @@ class AnalyticsEventNames {
 
```

Comment:
(rien à voir avec ce fichier)
Il manque l'event quand un utilisateur click sur une notification système (Android / iOS), qui est différente des notifications in-app : si l'app est en foreground et que l'app est reçue, on affiche une notification in-app.
Sinon c'est une notif système